### PR TITLE
Feature/elections improvements

### LINKED ISSUE #2204: Maps on campaign pages aren't rendering
In the election pages like:
https://www.fec.gov/data/elections/president/2016/
screen shot 2017-07-19 at 3 45 20 pm

if you click on the map you get an error
 console error "Cannot read property 'aDataSort' of undefined"

### PR DESCRIPTION
**Pull request by noahmanger**
This is some old work I started to re-do the elections pages in the new templates. It also contains the new comparison button for candidate pages, though needs the design from https://github.com/18F/openFEC-web-app/issues/2198 applied.

---
Update: this is ready now. I finished up the styling by pairing with @jenniferthibault , added the compare button to the candidate page, and also fixed a bug where the maps on the comparison page weren't showing (resolves https://github.com/18F/openFEC-web-app/issues/2204).

image

image

image

image

### CODE REVIEW

File: openfecwebapp/templates/elections.html [hunk 1]
```diff
@@ -13,73 +12,106 @@
...
+            href="#section-1"
+            >Candidate financial totals</a>
+        </li>
+        <li class="side-nav__item" role="presentation">
+          <a
+            class="side-nav__link"
+            role="tab"
+            data-name="contributions"
+            tabindex="0"
+            aria-controls="panel3"
```
Comment (xtine) [inline openfecwebapp/templates/elections.html]: Is this supposed to be `panel2` (and next `panel3`)?

File: openfecwebapp/filters.py [hunk 2]
```diff
@@ -85,3 +85,7 @@ def get_max(list):
 @app.template_filter()
 def get_min(list):
     return min(list)
+
+@app.template_filter()
+def strip_zero_pad(number):
```
Comment (xtine) [inline openfecwebapp/filters.py]: Would this template filter be better named as just `strip` if we need to strip other characters than zero padded numbers?

**Review by xtine — APPROVED** [review]
Looks good, just have a few minor questions.

File: openfecwebapp/templates/elections.html (hunk 1 above)
Comment (noahmanger) [inline openfecwebapp/templates/elections.html]: Fixed!

File: openfecwebapp/filters.py (hunk 2 above)
Comment (noahmanger) [inline openfecwebapp/filters.py]: Well this filter just strips out the `0` so I think that makes sense.

**Pull request merged**